tripal_layout/config/schema/tripal_layout.schema.yml

```
@@ -85,15 +85,51 @@ tripal_layout_field_group_item:
     label:
       type: string
       label: "The label of this field group"
+    label_visibility:
+      type: string
+      label: "Indicates if the label should be visable."
+    empty_label_behavior:
+      type: string
+      label: "Indicates how to handle an empty label"
+    region:
+      type: string
+      label: "Region to put the field group in (i.e. 'Content')"
```

should this be e.g. instead of i.e.?
```suggestion
      label: "Region to put the field group in (e.g. 'Content')"
```

---

I think both work but you are likely more correct so we'll go with that :-)